rules/coding-style/tests/Rector/Ternary/TernaryConditionVariableAssignmentRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\CodingStyle\Rector\Ternary\TernaryConditionVariableAssignmentRector\Fixture;
+
+function ternary()
+{
+    true ? $a = 1 : $a = 0;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\CodingStyle\Rector\Ternary\TernaryConditionVariableAssignmentRector\Fixture;
+
+function ternary()
+{
+    $a = true ? 1 : 0;
+}
+
+?>
```

Comment:
Could you also always provide a fixture that is skipped? So have `return null;` varians covered too

Reply:
Fixed